examples/custom-server-express/server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const next = require('next')
+const accepts = require('accepts')
+
+const app = next('.', { dev: true })
+const handle = app.getRequestHandler()
+
+app.prepare()
+.then(() => {
+  const server = express()
+
+  server.use((req, res, next) => {
```

Comment:
@rauchg I wish we could have remove this by handling accept types inside `app.render`.
Then we don't need any `server.use` at all.